### PR TITLE
Cart page and functionalities

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
# Added Cart page and card functionalities

### CODE REVIEW

File: src/pages/cart-page/CartPage.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  return (
+    <>
+      <Navigation />
+      <main>
+        <h2 className="text-center">MY CART</h2>
+        <div className="cart-page-wrapper">
+          <section className="cart-product">
+            {cartList.length > 0 &&
+              cartList.map((item) => (
```
Comment (Ankur9669) [inline src/pages/cart-page/CartPage.jsx:35]: You can try cartList?.map than you don't have to use this ternary opertaor to check whether list is empty or not

File: src/pages/wishlist-page/WishlistPage.jsx [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
     <>
       <Navigation />
 
       <main className="wishlist-page-main">
         <h3>All Products</h3>
-        <div className="product-container">
-          {productsList.map((item) => (
+        <div className="wishlist-container">
+          {wishlists.map((item) => (
             <CardVertical
```
Comment (Ankur9669) [inline src/pages/wishlist-page/WishlistPage.jsx]: Suggeestion: Use optional chaing in map

**Review by Ankur9669 — APPROVED** [review]
Most of the things look good to merge, nice conventions used

File: src/pages/cart-page/CartPage.jsx (hunk 1 above)
Comment (yaswanthmyneni) [reply src/pages/cart-page/CartPage.jsx:35]: i think optional chaining doesn't work for an empty array, as the initial state for cartList is [ ]

**Pull request merged**